api-tests/features/p2-coi-delete-identity.feature

```diff
@@ -0,0 +1,73 @@
+@Build
+Feature: Delete identity
+
+  Scenario: Fast Follow COI - Delete
```

Comment:
Can we avoid calling this 'Fast Follow' - it'll become meaningless over time. Perhaps `Update details failed - Account deletion` or similar?

(Same below)